translated: docs/developer-docs/integrations/rosetta

hokosugi opened the pull request:
疑問点・修正依頼点

index.md
* 88行目 「署名なしトランザクション」と「署名付きトランザクション」が何を指しているのかイマイチ不明です。
* 110, 116行目は原文が重複しています。
* 158行目 自己認証用とは、公開鍵からPrincipalが生成されましたよ、このPrincipalは公開鍵が元です、と強調していると思いますが、訳として必要かどうか確認お願いします。
* 182行目 `transfer`（以下のmint, burnも同じ）もコールの種類なので原文のままにしました。
* 218行目 「空のスタブ」という表現がしっくりきません。スタブはテストに近い表現とすると、空とはどういうことなのか、修正していただけると助かります。
* 228行目 `privileged`をどう訳したら迷いました。Mintのトランザクションは実行可能だと思われますが、どうやってTXを送信するかの問いなので、直訳の「特権的」では回答になっておらず「秘匿」であれば「そこまで教えられない」という意味で通じると思いそうしました。ただし、そもそも操作が秘匿されていない可能性もあり、そうであれば、間違えていると思われます。そもそも、この送金操作がメインネットなのかテストネットかで違ってくるようにも思われ、理解に及びませんでした。

hotkeys.md
* 37, 48, 51行目 16進法に（hex）を追加してあります。私が理解していないだけですが、16進法で、または反対の `HEX` だけで通じるのであれば、削除してください。

neuron-lifecycle.md
* 24行目 溶解状態（Dissolve State）は小見出しであるべきと思いますが、原文に従っています。
* 26行目 単語の統一でご相談です。`Age`を「エイジ」としましたが、どうでしょうか？複数箇所で両建てにしていますが、後半とその他のファイルでは「エイジ」となっています。
* 55行目 「ステートダイアグラム-v2」の本文は原文のままです。一応確認です。

staking-support.md
* 468, 522行目 `bounds included`を訳しておいたほうがよいでしょうか？１から100までと言えば100は含まれるので、強調ととればよいのですが、冗長にも思えます。
* 536行目 「投票中の followees の票から、フォローする Neuron の票を自動的に推定します。」 もっと良い表現はないかと思っていますが、このプルリク文を書いていて「それぞれの」をNeuronの頭に付けたら良いかも、と思いつきました。どうでしょうか？
* 625行目 「呼び出し側が選択した Neuron インデックスのリスト」というのかよく分かっていません。
* 
staking-tutorial.md
* 110行目 ここの`Notes:`は訳しても良かったのでしょうか？以前に指摘受けたのは 強調文内の `:::note` のことだったと思うのですが、未確認なのでそのままにしました。いま思うと、訳したほうが良かったかもと思っています。


このロゼッタに関するフォルダでは、イマイチ興味を引かないのと専門的なところが多くて翻訳に苦しみました。
間違ったところも多々あるかも知れません。修正あれば、都度訂正しますのでご指摘のみで結構です。よろしくおねがいします。

hokosugi committed:
translated: docs/developer-docs/integrations/rosetta

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+
+### Rosetta ノードをプロダクトモードで稼動する
+
+テストが終わったら、 `—interactive`、 `—tty`、`--rm` コマンドラインオプションを指定せずに、プロダクトモードで Docker イメージを実行する必要があります。
```
nitsです
```suggestion
テストが終わったら、 `--interactive`、 `--tty`、`--rm` コマンドラインオプションを指定せずに、プロダクトモードで Docker イメージを実行する必要があります。
```

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+
+テストが終わったら、 `—interactive`、 `—tty`、`--rm` コマンドラインオプションを指定せずに、プロダクトモードで Docker イメージを実行する必要があります。
+これらのコマンドラインオプションは、インタラクティブなターミナルセッションをアタッチし、コンテナを削除するために使用され、主にテスト目的のために用意されています。
+プロダクト環境で実行する場合は、`—detach` オプションを使用して Docker イメージを起動し、コンテナをバックグラウンドで実行し、オプションとしてブロックを保存するための `—volume` コマンドを指定します。
```
```suggestion
プロダクト環境で実行する場合は、`--detach` オプションを使用して Docker イメージを起動し、コンテナをバックグラウンドで実行し、オプションとしてブロックを保存するための `--volume` コマンドを指定します。
```

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+これらのコマンドラインオプションは、インタラクティブなターミナルセッションをアタッチし、コンテナを削除するために使用され、主にテスト目的のために用意されています。
+プロダクト環境で実行する場合は、`—detach` オプションを使用して Docker イメージを起動し、コンテナをバックグラウンドで実行し、オプションとしてブロックを保存するための `—volume` コマンドを指定します。
+
+Rosetta ノードインスタンスをメインネットに接続するためにフラグを追加します。`mainnet` と `--not-whitelisted` を追加します。
```
```suggestion
Rosetta ノードインスタンスをメインネットに接続するためにフラグを追加します。`--mainnet` と `--not-whitelisted` を追加します。
```

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+
+Docker イメージで提供される統合ソフトウェアには、Rosetta API の標準仕様にはない要件がひとつあります。
+
+ICP トークンを含むトランザクションでは、ネットワークが署名付きトランザクションを受け取る24時間以内に署名なしトランザクションが作成されなければなりません。
```
署名なしトランザクションという形でpayloadを構築し、そのpayloadに対して署名する２段階操作を行なうため、署名あり／なしの二種類のトランザクションがあるということのように見えます。
参考：
https://www.rosetta-api.org/docs/construction_api_introduction.html

```suggestion
ICP トークンを含むトランザクションでは、署名なしトランザクションはネットワークが署名付きトランザクションを受け取る24時間以内に作成されなければなりません。
```

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+
+- 最悪の場合、５分でタイムアウトです。
+
+- `mempool` API を使用しないでください。私たちの実装は空のスタブです。
```
空というのはインターフェースだけで実際のビジネスロジックが何も含まれていないという意味で、「実装が空」という言い方はよくしますので、問題ないと思います。「私たちの実装は〜」というのが直訳気味なので、少し修正しました
```suggestion
- `mempool` API を使用しないでください。空のスタブ実装となっています。
```

hoosan commented on docs/developer-docs/integrations/rosetta/index.md:
```diff
@@ -1,5 +1,256 @@
+
+#### Mint や Burn のトランザクションを送信するにはどうすればよいですか？
+
+- Mint は秘匿された操作です。現在、Rosetta API コールによる Burn はサポートしていません。
```
システムに対する高いレベルのアクセス権が必要なことを「特権」と日本語で呼んでいるので、ここでも特権で問題ないと思いますが、わかりやすさのため括弧書きとしました。秘匿はされていないと思います。
```suggestion
- Mint は高いレベルのアクセス権（特権）が必要な操作です。現在、Rosetta API コールによる Burn はサポートしていません。
```

hoosan commented on docs/developer-docs/integrations/rosetta/transfers.md:
```diff
@@ -1,3 +1,118 @@
+
+-   `TRANSACTION` タイプのオペレーションがアドレス `A` に `-T` の値で適用されます。
+
+-   `TRANSACTION` オペレーションタイプがアドレス `B` に金額 `T` で適用されます。
```
統一観点です
```suggestion
-   `TRANSACTION` タイプの操作がアドレス `B` に金額 `T` で適用されます。
```

hoosan commented on docs/developer-docs/integrations/rosetta/transfers.md:
```diff
@@ -1,3 +1,118 @@
+
+送金額 `T` をアドレス `A` からアドレス `B` に転送するトランザクションは、3つのオペレーションを含むことになります：
+
+-   `TRANSACTION` タイプのオペレーションがアドレス `A` に `-T` の値で適用されます。
```
統一観点です
```suggestion
-   `TRANSACTION` タイプの操作がアドレス `A` に `-T` の値で適用されます。
```

hoosan commented on docs/developer-docs/integrations/rosetta/transfers.md:
```diff
@@ -1,3 +1,118 @@
+
+トランザクション内の操作の順序は関係ありません。
+
+1回の取引で複数回の送金はできません。そのような取引の結果は明示されません。
```
unspecifiedからは、そもそも定義されていないというニュアンスを感じます
```suggestion
1回のトランザクションで複数の送金を行うことはできません。そのような取引の結果は未定義です。
```

hoosan commented on docs/developer-docs/integrations/rosetta/transfers.md:
```diff
@@ -1,3 +1,118 @@
+
+-   アドレス `A` は、サブアカウントがトランザクションの署名に使用した公開鍵に由来する Principal になっています。
+
+-   `FEE` オペレーションで指定する金額は `suggested_fee` と絶対値で等しくなっています。
```
条件の箇条書きなので「〜していること」とした方が良いと思いました
```suggestion
-   アドレス `A` は少なくとも `T` + `suggested_fee` の ICP を保持していること。

-   アドレス `A` は、サブアカウントがトランザクションの署名に使用した公開鍵に由来する Principal になっていること。

-   `FEE` オペレーションで指定する金額は `suggested_fee` と絶対値で等しくなっていること。
```

hoosan commented on docs/developer-docs/integrations/rosetta/transfers.md:
```diff
@@ -1,3 +1,118 @@
+
+### 例
+
+以下は、アドレス `bdc4ee05d42cd0669786899f256c8fd7217fa71177bd1fa7b9534f568680a938` からアドレス `b64ec6f964d8597afa06d4209dbce2b2df9fe722e86aeda2351bd95500cf15f8` まで 1 ICP を譲渡した取引のサンプルです。
```
```suggestion
以下は、アドレス `bdc4ee05d42cd0669786899f256c8fd7217fa71177bd1fa7b9534f568680a938` からアドレス `b64ec6f964d8597afa06d4209dbce2b2df9fe722e86aeda2351bd95500cf15f8` まで 1 ICP を譲渡したトランザクションのサンプルです。
```

hoosan commented on docs/developer-docs/integrations/rosetta/neuron-lifecycle.md:
```diff
@@ -1,3 +1,70 @@
+
+-   任意の台帳のアカウントから、ある** ICP 数量**を Neuron のアドレスに転送します。台帳上のNeuron アドレスはガバナンス Canister に属しています。呼び出し側は、2つの部分から Neuron アドレスを計算します。
+
+    -   ガバメント Canister の Principal。
```
```suggestion
    -   ガバナンス Canister の Principal。
```

hoosan commented on docs/developer-docs/integrations/rosetta/neuron-lifecycle.md:
```diff
@@ -1,3 +1,70 @@
+DISSOLVED  
+Neuron の溶解遅延は 0 になっています。Neuron ホルダーは、張り付けた ICP を払い出すか、溶解遅延を増加させるか、自由に行うことができ、これにより **NOT_DISSOLVING** 状態へ遷移します。
+
+エイジ（Age）  
```
エイジに異論ありません

hoosan commented on docs/developer-docs/integrations/rosetta/neuron-lifecycle.md:
```diff
@@ -1,3 +1,70 @@
+
+## 状態遷移
+
+    ステートダイアグラム-v2
```
stateDiagram と一つの変数名のようになっているので、原文通りで良いと思います
```suggestion
    stateDiagram-v2
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+# ステーキングとNeuron 管理
+
+本ドキュメントは、Rosetta API を拡張し、Internet Computer 上での資金  “Neuron” のステーキングとガバナンス “Neuron” の管理を可能にすることを規定しています。
```
nitsです
```suggestion
本ドキュメントは、Internet Computer 上での資金のステーキングとガバナンス “Neuron” の管理を可能にする Rosetta API の拡張について規定しています。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+:::
+
+## Neuron アドレスの導出する
```
```suggestion
## Neuron アドレスの導出
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
````diff
@@ -1,3 +1,819 @@
+}
+```
+
+## ステークファンド
````
```suggestion
## 資金のステーク
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   このとき、Neuron は `DISSOLVING` 状態です。
```
```suggestion
-   Neuron が `DISSOLVING` 状態である。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は Neuron コントローラーの台帳アドレスです。
```
事前条件、事後条件は「どういう状態であるか」を示すのが良いと思います。
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   Neuron は `NOT_DISSOLVING` 状態です。
```
```suggestion
-   Neuron が `NOT_DISSOLVING` 状態である。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
pre-postを対応づけるため、事前条件、事後条件とするのはどうでしょうか
（以降同様です）
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
+
+-   ホットキーは Neuron にリンクされています。
```
```suggestion
-   ホットキーが Neuron にリンクされている。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
+
+-   親 Neuron は少なくとも 1 ICP 相当の成熟度を持ちます。
```
```suggestion
-   親 Neuron は少なくとも 1 ICP 相当の成熟度を持つ。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+-   親 Neuron の成熟度は `0` に設定される。
+
+-   新しい Neuron は、転送された成熟度と同じ残高で生成されます。
```
```suggestion
-   新しい Neuron が、転送された成熟度と同じ残高で生成される。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `account.address` は、Neuron コントローラーの台帳アドレスです。
```
```suggestion
-   `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   成熟度はマージにより減少します。
```
```suggestion
-   成熟度がマージにより減少する。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+-   成熟度はマージにより減少します。
+
+-   マージ後、Neuron のステーキング数は増加します。
```
```suggestion
-   マージ後、Neuron のステーキング数が増加する。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+* `account.address` は、Neuron コントローラーまたはホットキーの台帳アドレスです。
```
```suggestion
* `account.address` が Neuron コントローラーまたはホットキーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+</div>
+
+* `account.address` は、Neuron コントローラーまたはホットキーの台帳アドレスです。
+* `metadata.followees` は、有効な Neuron 識別子の配列です。
```
```suggestion
* `metadata.followees` が有効な Neuron 識別子の配列である。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+* `account.address` は、Neuron コントローラーまたはホットキーの台帳アドレスです。
+* `metadata.followees` は、有効な Neuron 識別子の配列です。
+* `metadata.topic` は、有効なトピック識別子です。
```
```suggestion
* `metadata.topic` が有効なトピック識別子である。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+* Neuron は指定されたフォロールールに従って投票します。
```
```suggestion
* Neuron が指定されたフォロールールに従って投票する。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+<div class="formalpara-title">
+
+**前提条件：**
```
```suggestion
**事前条件：**
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+</div>
+
+-   `public_key` には、Neuron のコントローラーの公開鍵が含まれます。
```
```suggestion
-   `public_key` に、Neuron のコントローラーの公開鍵が含まれている。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
````diff
@@ -1,3 +1,819 @@
+}
+```
+
+- `account.address` は Neuron コントローラーの台帳アドレスです。
````
```suggestion
- `account.address` が Neuron コントローラーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
````diff
@@ -1,3 +1,819 @@
+}
+```
+
+- `account.address` は、neuron ホットキーの台帳アドレスです。
````
```suggestion
- `account.address` が neuron ホットキーの台帳アドレスである。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
````diff
@@ -1,3 +1,819 @@
+```
+
+- `account.address` は、neuron ホットキーの台帳アドレスです。
+- `metadata.controller.public_key` は、Neuron コントローラーの公開鍵です。
````
```suggestion
- `metadata.controller.public_key` が Neuron コントローラーの公開鍵である。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+2.  [curl](https://curl.se/)
+
+3.  ed25519 キーと、そのキーを使ってメッセージに署名するための「お気に入りのツール」。
```
原文は特に強調する表現になっていないので、「」はなくても良いかと思いました
```suggestion
3.  ed25519 キーと、そのキーを使ってメッセージに署名するためのお気に入りのツール。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+# Rosetta ステーキングのチュートリアル
+
+このドキュメントでは、[Rosetta API](https://rosetta-api.org) のインターネットコンピュータ実装を使用して Neuron を作成する手順を説明します。ガバナンス Canister への資金移動、 Neuron を作成し、新しく作成された Neuron を設定します。
```
```suggestion
このドキュメントでは、[Rosetta API](https://rosetta-api.org) に関する Internet Computer の実装を使用して Neuron を作成する手順を説明します。ガバナンス Canister への資金移動、 Neuron を作成し、新しく作成された Neuron を設定します。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+## 概要
+
+インターネットコンピュータから見ると、 Neuron を作るのは多段階の操作になります。
```
```suggestion
Internet Computer から見ると、 Neuron を作るのは多段階の操作になります。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+2.  ガバナンスに送金を通知します。このステップで、ガバナンス Canister は新しい Neuron を割り当てます。
+
+3.  新しく作成された Neuron を設定する：溶解遅延を設定し、溶解を開始します。
```
```suggestion
3.  新しく作成された Neuron に対して、溶解遅延と溶解を開始する設定を行います。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+Rosetta API では、これらのステップをトランザクションの [operation](https://www.rosetta-api.org/docs/models/Operation.html) として表現しています。これらの操作の実行は、[Construction API flow](https://www.rosetta-api.org/docs/flow.html#construction-api) に従わなければなりません。
+
+## アカウント識別子の導出する
```
```suggestion
## アカウント識別子を導出する
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+## 署名するペイロードを取得する
+
+Notes:
```
Notes:　は訳した方が良いと思いました
```suggestion
以下の点を注意してください：
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-tutorial.md:
```diff
@@ -1,3 +1,801 @@
+
+## ペイロードに署名し、署名付きトランザクションを構築する
+
+お好みのツールを使って、 `/construction/payloads` レスポンスにあるペイロードに署名してください。以下のコマンドで示すように、署名を `/construction/combine` リクエストに含めます。
```
nitsです
```suggestion
お好きなツールを使って、 `/construction/payloads` レスポンスにあるペイロードに署名してください。以下のコマンドで示すように、署名を `/construction/combine` リクエストに含めます。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+- メタデータフィールド `controller` はオプションで、デフォルトでは既存の Neuron コントローラーと同じになります。
+
+- メタデータフィールド `percentage_to_spawn` はオプションで、デフォルトでは 100 となります。指定する場合は、1 から 100 までの整数値（境界を含む）である必要があります。
```
boundsは原文に従って訳すのが良いと思いました。境界値を含む、とするのはどうでしょうか。
```suggestion
- メタデータフィールド `percentage_to_spawn` はオプションで、デフォルトでは 100 となります。指定する場合は、1 から 100 までの整数値（境界値を含む）である必要があります。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+
+`FOLLOW` 操作は、Neuron に対してフォロールールを設定します。
+ガバナンス Canister スマートコントラクトは、投票中の followees の票から、フォローする Neuron の票を自動的に推定します。
```
foloweesとすると役割がわかりにくいので、少し修正しました
```suggestion
ガバナンス Canister スマートコントラクトは、投票中、フォローされている Neuron の票から、その Neuron をフォローしている Neuron の票を自動的に導出します。
```

hoosan commented on docs/developer-docs/integrations/rosetta/staking-support.md:
```diff
@@ -1,3 +1,819 @@
+
+:::note
+
+`followees` メタデータフィールドには、ガバナンス Canister スマートコントラクトによって割り当てられた一意の Neuron 識別子のリストが含まれます。これは呼び出し側が選択した Neuron インデックスのリストではありません。
```
neuron_index というフィールドがあるので、この値ではないと言っているのではないかと思います。

hoosan reviewed (changes requested):
翻訳ありがとうございます。Rosetta-API周りはICで送金関係を実装しようとすると避けて通れないところで、かつ理解が難しい内容ですので、ドキュメントの訳版があるというのは価値が高いと思います。自分もあまり読み込んでいなかったので勉強になりました。

hokosugi committed:
Apply suggestions from code review

レビューありがとうございました。
異論がないのですべてコミットします。
コマンドラインオプションのところは、日頃から使っていない事がバレたようで気恥ずかしいｗ

Co-authored-by: hoosan <40290137+hoosan@users.noreply.github.com>
